CONTRIBUTORS

```diff
@@ -10,14 +10,27 @@
 # Please keep each section sorted alphabetically!
 
 SIG-archinfra:
+    - annajung
     - chinhuang007
+    - chudegao
     - faxu
+    - fdwr
+    - gwang-msft
+    - jackwish
+    - jantonguirao
     - kevinch-nv
+    - mhamilton723
+    - rajeevsrao
     - skottmckay
+    - snnn
     - spandantiwari
+    - stillmatic
+    - take-cheeze
+    - TomWildenhain-Microsoft
```

Comment:
I don't expect them to continue to contribute

Reply:
updated to not adding them.